shoes-core/spec/shoes/shared_examples/dsl/rgb.rb

```diff
@@ -5,7 +5,8 @@
   let(:alpha) { 133 } # cornflowerblue
 
   it "sends args to Shoes::Color" do
-    expect(Shoes::Color).to receive(:new).with(red, green, blue, alpha)
+    expect(Shoes::Color).to receive(:new).ordered
```

Comment:
You could go `allow(Shoes::Color).to receive ...`

Then after you did whatever you wanted to do you can do: `expect(Shoes::Color).to have_received(:something).exactly(n).times` (there is also at_least and others)